kong/plugins/prometheus/exporter.lua

```diff
@@ -27,25 +28,30 @@ local function init()
   -- per service
   metrics.status = prometheus:counter("http_status",
                                       "HTTP status codes per service in Kong",
-                                      {"code", "service"})
+                                      {"code", "service", PIVOT_LABEL_NAME})
   metrics.latency = prometheus:histogram("latency",
                                          "Latency added by Kong, total request time and upstream latency for each service in Kong",
-                                         {"type", "service"},
+                                         {"type", "service", PIVOT_LABEL_NAME},
                                          DEFAULT_BUCKETS) -- TODO make this configurable
   metrics.bandwidth = prometheus:counter("bandwidth",
                                          "Total bandwidth in bytes consumed per service in Kong",
-                                         {"type", "service"})
+                                         {"type", "service", PIVOT_LABEL_NAME})
```

Comment:
@jmdacruz Didn't we discuss that the labels be dynamic based on the configuration?
It seems like this will result in a static `pivot` label in this case.

Reply:
@hbagdi I haven't been able to work on this yet, this was just the initial attempt to get the ball rolling on the dev env. I'll have to see if I can spend some time on this soon

Reply:
Yes, dynamic labels would be really really helpful.